### ISSUE TITLE
OpenRefine 3.8.6 incorrectly says it needs updating even when it's the latest version

### ISSUE THREAD

**Issue by vivian-rook** [OpenRefine/OpenRefine#7001]
I install OpenRefine into an ubuntu container to deploy as part of paws (https://wikitech.wikimedia.org/wiki/PAWS) I preformed the following change to update to 3.8.6:
```
 ENV OPENREFINE_DIR /srv/openrefine
 ENV PATH=$PATH:$OPENREFINE_DIR
 RUN mkdir -p ${OPENREFINE_DIR} && cd ${OPENREFINE_DIR} && \
-    curl -L 'https://github.com/OpenRefine/OpenRefine/releases/download/3.8.5/openrefine-linux-3.8.5.tar.gz' | tar xzf - --strip=1
+    curl -L 'https://github.com/OpenRefine/OpenRefine/releases/download/3.8.6/openrefine-linux-3.8.6.tar.gz' | tar xzf - --strip=1

```

This seemed to build fine, though when I started to container I still saw OpenRefine listing 3.8.5 as its version with the update banner. I do see in the licenses.xml:
```
      <groupId>org.openrefine</groupId>
      <artifactId>server</artifactId>
      <version>3.8.6</version>
```
Suggesting to me that it updated. Though it doesn't seem to be fully updated. Is this install updating to the next version?

### To Reproduce
1. First, do ...
2. Then, ...
3. Finally, ...

### Versions
 - Operating System: 
 - Browser Version: 
 - JRE or JDK Version: 
 - OpenRefine:

**Comment by wetneb** [comment]
Thanks for reporting! It looks like the commit that changed the version (https://github.com/OpenRefine/OpenRefine/commit/c1f11d07e4ce52aa1c6ff1e36701bc5833f47ab2) did not update RefineServlet.java, which explains that.
cc @tfmorris

**Comment by tfmorris** [comment]
It looks like the Maven command in our release [instructions](https://openrefine.org/docs/technical-reference/version-release-process) didn't update that particular copy of the version string. For last release I updated all the versions by hand, so didn't encounter this issue.

I'll spin a new release.

**Comment by tfmorris** [comment]
This should be fixed in 3.8.7
